js/src/dapps/githubhint/Application/application.js

```diff
@@ -128,11 +131,13 @@ export default class Application extends Component {
               <Button
                 disabled={ registerBusy }
                 invert={ registerType !== 'file' }
-                onClick={ this.onClickTypeNormal }>File Link</Button>
+                onClick={ this.onClickTypeNormal }
+              >File Link</Button>
```

Comment:
I know the diff has been produced automatically, but this is still not consistent with the other places. Consider adapting it. – Same below.